### DISCUSSION TITLE
How can I replace to ‘chrome’ url scheme

### DISCUSSION THREAD

**Discussion by taiseiue** [cefsharp/CefSharp#4143] [Q&A] [answered]
I want to replace ‘chrome://’ url scheme.like MS-Edge’s ‘edge://’ , Opera ‘s ‘opera://’,etc…
Are there any ideas?

**Comment by amaitland** [comment, marked as answer]
Far as I'm aware they patch chromium to customize the behaviour. There's no configuration option.

**Reply by taiseiue** [reply]
Thank you for your answer.So,could you tech me how to patch chromium to customize the behaviour?

**Reply by amaitland** [reply]
You'd first need to build CEF from source. 

https://bitbucket.org/chromiumembedded/cef/wiki/MasterBuildQuickStart.md

Then use it's patching mechanism see https://github.com/chromiumembedded/cef/tree/master/patch

CEF has it's own support forum at https://magpcss.org/ceforum/index.php

As it exactly what chromium changes are required that's on you to research. Perhaps there is another open source project that's already made the changes, see how they did it.

That's all the guide I can offer, best of luck.
